src/TLabel.cpp

```diff
@@ -87,88 +92,84 @@ void TLabel::mousePressEvent(QMouseEvent* event)
 {
     if (forwardEventToMapper(event)) {
         return;
-    }
-    if (mMouseButtons.contains(event->button())) {
-        if (mpHost) {
-            TEvent tmpClickParams = mClickParams;
-            tmpClickParams.mArgumentList.append(mMouseButtons.value(event->button()));
-            tmpClickParams.mArgumentTypeList.append(ARGUMENT_TYPE_STRING);
-            mpHost->getLuaInterpreter()->callEventHandler(mClick, tmpClickParams);
-        }
+    } else if (mpHost && mClick != "") {
```

Comment:
🌍 __I18n__ Please avoid doing checks for empty/non-empty `QString`s with a comparison to a raw const char literal - including the empty string! I suggest using the `QString::isEmpty()`  (or `QString::isNull()` which is similar but subtly different, _RTF Qt Docs_  😈) so there are less raw strings to wrap in `tr(`...`)`/`translate(`...`)`/`QLatin1String(`...`)` or `QStringLiteral(`...`)` later for __Mudlet 4.x__ ....

Reply:
So change this to `mpHost && !mClick.isEmpty()`, reads better too!